scripts/plugins/openvswitch-config-update

```diff
@@ -114,187 +110,232 @@ def update(session, args):
         query = 'field "management"="true"'
         recs = session.xenapi.PIF.get_all_records_where(query)
         for rec in recs.itervalues():
-            pool_mgmt_macs[rec.get('MAC')] = rec.get('device')
+            pool_mgmt_macs[rec.get("MAC")] = rec.get("device")
 
     dib_changed = False
     fail_mode_changed = False
-    for bridge in vswitchCfgQuery(['list-br']).split():
-        if not bridge in bton:
+    for bridge in vswitchCfgQuery(["list-br"]).split():
+        if bridge not in bton:
             continue
         network = bton[bridge]
-        bridge = vswitchCfgQuery(['br-to-parent', bridge])
+        bridge = vswitchCfgQuery(["br-to-parent", bridge])
 
-        xapi_dib = network['other_config'].get('vswitch-disable-in-band')
+        xapi_dib = network["other_config"].get("vswitch-disable-in-band")
         if not xapi_dib:
-            xapi_dib = ''
-
-        ovs_dib = vswitchCfgQuery(['--', '--if-exists', 'get', 'Bridge',
-                                   bridge,
-                                   'other_config:disable-in-band']).strip('"')
+            xapi_dib = ""
+
+        ovs_dib = vswitchCfgQuery(
+            [
+                "--",
+                "--if-exists",
+                "get",
+                "Bridge",
+                bridge,
+                "other_config:disable-in-band",
+            ]
+        ).strip('"')
 
         # Do nothing if setting is invalid, and warn the user.
-        if xapi_dib not in ['true', 'false', '']:
-            ret_str += '"' + xapi_dib + '"' + \
-                ' is an invalid value for vswitch-disable-in-band on ' + \
-                bridge + '  '
+        if xapi_dib not in ["true", "false", ""]:
+            ret_str += (
```

Comment:
This looks unidiomatic; why not using string formatting to create a single string that is added to `ret_str`? Again, it is existing code.

Reply:
Indeed, I tried to use f-string, but it led to build failures whilst py_byte_compile running (https://docs.fedoraproject.org/en-US/packaging-guidelines/Python/#_source_files_and_bytecode_cache), it need to be compatible with both py2 and py3 in the current xapi.spec.

If you agreed, I can use classic string format with the % operator, even thought its performance is not as good as f-string in py3.